src/ex5/server/clients/pokemon_client.js

```diff
@@ -1,23 +1,24 @@
-import axios from "axios";
+const axios = require("axios");
 const url = "https://pokeapi.co/api/v2/pokemon";
 let requestsArray = [];
+const TWO_MINS = 2;
 
-export async function fetchPokemon(pokemonId) {
+async function fetchPokemon(pokemonId) {
   const date = new Date();
   let curretHour = parseInt(date.getHours());
   let curretMinute = parseInt(date.getMinutes());
   const isThisRequestExist = [...requestsArray].some((item) => {
-    if (item.id === pokemonId) {
-      if (item.hour === curretHour) {
-        if (curretMinute - item.minute < 2) {
-          return true;
-        }
-      }
+    if (
+      item.id === pokemonId &&
+      item.hour === curretHour &&
+      curretMinute - item.minute < TWO_MINS
+    ) {
+      return true;
     }
   });
   if (isThisRequestExist) {
     return {
-      name: `Request overload with id: ${pokemonId}:please wait a few min before trying again`,
+      name: `Request overload with id: ${pokemonId}:please wait 2 minutes before trying again`,
```

Comment:
```suggestion
      name: `Request overload with id: ${pokemonId}:please wait ${NUMBER_OF_MINUTES} minutes before trying again`,
```
use the const name so if you change the value you need to only do this in 1 place